### ISSUE TITLE
[Bug Report] Bible metadata desn't get imported

### ISSUE THREAD

**Issue by yurividal** [ChurchApps/FreeShow#902]
I'm trying to set Copyright metadata to an entire bible.
I imported it from OpenSong XML, and i noticed that the opensong xml doesn't include a metadata field.

So, i found the .fsb file of the bible that i had just imported, made a copy of it, added the copyright informatin, and re-imported that into Freeshow, as a new bible. This time, the copyright metadata is in the file, but it stil doesn't import the metadata properly.

I believe the code to detect and import metadata has to be added to the file /src/frontend/converters/bible.ts

**Comment by vassbo** [comment]
This is a feature I guess. How would you need the metadata to be displayed in the program?

At the moment only a `copyright` key is allowed, and it will show at the bottom of the bible verses.
I think a `metadata` key with any values like this could be the way to go, `{ "publisher": "" }`

**Comment by yurividal** [comment]
I am trying to display a copyright information of the Bible on the bottom of every slide.

I have the {metadata_copyright} tag in my template, and it works. But I am having to manually put it in every time I add a scripture slide.

I tried editing the .fsb too, but when I close and re-open freeshow it gets overwritten and the copyright meta is gone.

**Comment by vassbo** [comment]
I'll add a toggle to display metadata simular to how show metadata works.

You need to close the program first, then edit the .fsb, then reopen.

**Comment by yurividal** [comment]
I dont think you need to add this toggle. As I said, I'm already able to display it using the {metadata_copyright}. 
What currently is not possible to do is to SET a copyright value to the entire Bible database.

**Comment by yurividal** [comment]
I closed freeshow, edited the file C:\Users\yuri\Documents\FreeShow\Bibles\ESV.fsb to look like this:

````
["1f722590649",{"name":"ESV","copyright":"test","books":[{"name":"Genesis","number":1,"chapters":[{"number":"1","verses":[{"number":"1","value":"In the beginning, God created the heavens and the earth."},{"number":"2","value":"The earth was without form and void, and darkness was over the face of the deep. And the Spirit of God was hovering over the face of the waters."},{"number":"3","value":"And God said, “Let there be light,” and there was light."},{"number":"4","value":"And God saw that the light was good.
````

Then, open frewshow again, add a new scripture show for Genesis 1, and still, it doesn't inherit the "test" in the copyright field.

![image](https://github.com/user-attachments/assets/f02f9895-d6a1-413f-a4e8-1ee81114b6b9)

**Comment by yurividal** [comment]
> I guess I only need to move any scripture metadata to the show when creating a show from a scripture.

I think this is what is missing.

**Comment by yurividal** [comment]
> Metadata from Bible file will be moved to show if available in this format (any key supported):
> 
> ```json
> {"name":"ESV","metadata":{"title": "", "publisher": "", "copyright": ""},"books": ...
> ```

Hi @vassbo , the metadata is still not being shown in the scriptures.
![image](https://github.com/user-attachments/assets/ab2c3c35-3dfe-4f43-b6c9-195811d41c81)


And here is the start of my bible .fsb
````
["1f722590649",{"name":"ESV","metadata":{"title":"2002 English Standard Version: Anglicized Edition (ESVUK)","publisher":"HarperCollins","copyright":"HarperCollins Religious"},"books":[{"name":
...
````

**Comment by yurividal** [comment]
I have tried re-importing the fsb file, but still no metadata in the slides

**Comment by yurividal** [comment]
> You don't need to re-import the .fsb, but you have to click "New show" again. That should work, at least it did for me just now!

No, it's not working, It worked for 2 times. i added 2 new readings, and the metadata was there. Then, i added one more, and its not there any more. 

https://github.com/user-attachments/assets/d10f66ba-bd04-4099-ba1a-b45b6ef3201d

**Comment by yurividal** [comment]
Ok, i think i found where the bug is. It works until you select a template.
In the video bellow, i will add a scripture. meta data information comes just fine.  notice how the template (which should have the metadata in the bottom) is not properly applied, so i apply the template again, and the metadata appears. From this moment on, whenever i add a new scripture, metadata comes blank.


https://github.com/user-attachments/assets/fdd37cf4-aff5-46e4-b463-a77d931d3049
